lib/sycamore/sycamore/transforms/extract_entity.py

```diff
@@ -1,14 +1,14 @@
 from abc import ABC, abstractmethod
 from typing import Callable, Any, Optional, Union
 
-
+from sycamore import context
```

Comment:
this feels weird to me, not sure why. Think I'd be more comfortable with either using it as `sycamore.context.current()` or exporting current to sycamore and having `sycamore.current()`. Eh, I guess this is fine. Disregard the above unless you want to.

Reply:
I think the former is fine, little more readable too - will do